### PR TITLE
Use random UUID string as boundary instead of hardcoded '__END_OF_PAR…

### PR DESCRIPTION
**Pull request by dmitry-fa**
issue 909

### CODE REVIEW

**Comment by rahulKQL** [comment]
Have you run `mvn com.coveo:fmt-maven-plugin:format` with this change? If not then please do.

**Comment by dmitry-fa** [comment]
Yes, I ran this command. Build successful.

**Review by rahulKQL — APPROVED** [review]
LGTM

File: google-http-client/src/test/java/com/google/api/client/http/MultipartContentTest.java [hunk 1]
```diff
@@ -26,55 +26,76 @@
...
-            + CRLF
-            + "--myboundary--"
-            + CRLF,
+        "--myboundary" + CRLF
+            + HEADERS + CRLF
+            + "foo" + CRLF
+            + "--myboundary" + CRLF
+            + HEADERS + CRLF
+            + "bar" + CRLF
+            + "--myboundary--" + CRLF,
```
Comment (rahulKQL) [inline google-http-client/src/test/java/com/google/api/client/http/MultipartContentTest.java:98]: After running that build I believe these files get updated automatically, If there is any modification after build then please add them as well.

**Pull request closed without merging**